docs/build/local-first.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_label: Local-first cache
+sidebar_position: 1.5
+description: "Get started building apps with XMTP."
+---
+
```

Comment:
Added this as a standalone doc at the top of the Build section. In general, this is the first thing devs need to be aware of before they start building, correct?

Reply:
i think that makes sense